AK/Error.h

```diff
@@ -37,6 +37,13 @@ class Error {
         return from_string_view(StringView { string_literal, N - 1 });
     }
 
+    // Note: Don't call this from C++; it's here for Jakt interop.
+    template<SameAs<StringView> T>
+    ALWAYS_INLINE static Error from_string_literal(T string)
```

Comment:
maybe it should be name it accordingly to discourage the use? 🤔
e.g. `__jakt__from_string_literal` `__INTERNAL__from_string_literal` or similarly? (same in other places)

Reply:
The name must be _exactly_ that I'm afraid (Jakt generates Foo::from_string_literal)

Reply:
Oh shit sorry, this is the Error one
Yeah I guess we can name it weirdly